### PR TITLE
Employee Salary Update

### PR DESCRIPTION
**Pull request by O-Johnson**
Okwori Johnson

### CODE REVIEW

File: O-Johnson/employeeSalaryUpdate.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+let Johnson = employeeInfo('Johnson', 'backend developer', 5000);
+
+const basicSalary = (employee) => {
+  return `The basic salary of ${employee.name} for the role of ${employee.role} is ${employee.salary} per month`;
+};
+
+// basic salary before promotion
+console.log(basicSalary(Johnson));
+
+const salaryUpdate = (employee) => {
```
Comment (joshuaunity) [inline O-Johnson/employeeSalaryUpdate.js:14]: Hi, it seems the function isn't producing correct results, If the initial salary is 5000 and now want to change it to 3000, wth your function the employee will end up getting 8000. Please double-check the code and run thorough tests to ensure it functions correctly in all scenarios.

Comment (O-Johnson) [reply O-Johnson/employeeSalaryUpdate.js:14]: The initial salary as a backend developer is 5000. After promotion, 3000 gets added to the initial salary (employee.salary + 3000). This takes the updated salary after promotion to 8000.

Comment (joshuaunity) [reply O-Johnson/employeeSalaryUpdate.js:14]: you still don't get the point. I don't want to add to the salary, I want to update the main salary to whatever I want. if I put 1000 then the salary should be changed to 1000 and not 1000 to be added to the salary.

Comment (joshuaunity) [reply O-Johnson/employeeSalaryUpdate.js:14]: if this was a promotion feature then this is fine but its not its a salaryupdate feature

Comment (O-Johnson) [reply O-Johnson/employeeSalaryUpdate.js:14]: okay, noted.

**Pull request merged**